bundles/binding/org.openhab.binding.zwave/database/philio/pst02.xml

```diff
@@ -0,0 +1,235 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+	<Model>PST02</Model>
+	<Label lang="en">Slim Multi-Sensor</Label>
+	<CommandClasses>
+		<Class><id>0x20</id></Class>	<!-- COMMAND_CLASS_BASIC -->
+		<Class><id>0x30</id></Class>	<!-- COMMAND_CLASS_SENSOR_BINARY_V2 -->
+		<Class><id>0x31</id></Class>	<!-- COMMAND_CLASS_SENSOR_MULTILEVEL_V5 -->
+		<Class><id>0x59</id></Class>	<!-- COMMAND_CLASS_ASSOCIATION_GRP_INFO -->
+		<Class><id>0x5A</id></Class>	<!-- COMMAND_CLASS_DEVICE_RESET_LOCALLY -->
+		<Class><id>0x5E</id></Class>	<!-- COMMAND_CLASS_ZWAVEPLUS_INFO_V2 -->
+		<Class><id>0x70</id></Class>	<!-- COMMAND_CLASS_CONFIGURATION -->
+		<Class><id>0x71</id></Class>	<!-- COMMAND_CLASS_NOTIFICATION_V4 -->
+		<Class><id>0x72</id></Class>	<!-- COMMAND_CLASS_MANUFACTURER_SPECIFIC_V2 --> 
+		<Class><id>0x73</id></Class>	<!-- COMMAND_CLASS_POWERLEVEL -->
+		<Class><id>0x7A</id></Class>	<!-- COMMAND_CLASS_FIRMWARE_UPDATE_MD_V2 -->
+		<Class><id>0x80</id></Class>	<!-- COMMAND_CLASS_BATTERY -->
+		<Class><id>0x84</id></Class>	<!-- COMMAND_CLASS_WAKE_UP_V2 -->
+		<Class><id>0x85</id></Class>	<!-- COMMAND_CLASS_ASSOCIATION_V2-->
+		<Class><id>0x86</id></Class>	<!-- COMMAND_CLASS_VERSION_V2 -->
+		<Class><id>0x8F</id></Class>	<!-- COMMAND_CLASS_MULTI_CMD -->
+		<Class><id>0x98</id></Class>	<!-- COMMAND_CLASS_SECURITY -->
+		<Class><id>0xEF</id></Class>	<!-- COMMAND_CLASS_MARK -->
+	</CommandClasses>
+	<Configuration>
+		<Parameter>
+			<Index>2</Index>
+			<Type>byte</Type>
+			<Default>-1</Default>
+			<Minimum>-1</Minimum>
```

Comment:
Ok - let's leave it like it is. I'm just worried this will cause problems - I know it has in the past, but I can't remember what the problem was now.